cmd/cli/serve/ucan.go

```diff
@@ -313,9 +313,16 @@ func startServer(cmd *cobra.Command, _ []string) error {
 
 	defer svc.Close(ctx)
 
-	presolv, err := principalresolver.New(cfg.ServicePrincipalMapping)
+	presolv, err := principalresolver.NewHTTPResolver(map[string]string{
+		indexingServiceDID.String(): indexingServiceDID.String(),
+		uploadServiceDID.String():   uploadServiceDID.String(),
```

Comment:
Not sure I understand why this is passed in here? Is this an allowlist?

I think the values are meant to be URL strings?

In theory the DID document you're meant to lookup is derived from the DID i.e. if you're asked to resolve `did:web:example.com`, you make a HTTP request to `https://example.com/.well-known/did.json`

Reply:
I've made this change in 12f6449fac04e674b8ce5816c325ccb01e55f57c